Sources/BetterSafariView/SafariView/SafariViewPresenter.swift

```diff
@@ -79,20 +79,31 @@ extension SafariViewPresenter {
             let safariViewController = SFSafariViewController(url: representation.url, configuration: representation.configuration)
             safariViewController.delegate = self
             representation.applyModification(to: safariViewController)
-            
-            // Present a Safari view controller from the `viewController` of `UIViewRepresentable`, instead of `UIViewControllerRepresentable`.
-            // This fixes an issue where the Safari view controller is not presented properly
-            // when the `UIViewControllerRepresentable` is detached from the root view controller (e.g. `UIViewController` contained in `UITableViewCell`)
-            // while allowing it to be presented even on the modal sheets.
-            // Thanks to: Bohdan Hernandez Navia (@boherna)
-            guard let presentingViewController = uiView.viewController else {
-                self.resetItemBinding()
+
+            if #available(iOS 17.0, *) {
```

Comment:
Great! I think it would be nice to use your suggestion even on iOS 16 and earlier.

This guarantees that `presentingViewController` doesn't have a `presentedViewController`, which means the modal presentation of`SafariViewController` will not be blocked by any other `presentedViewController`.